models/renalStructures_UNEST_segmentation/configs/metadata.json

```diff
@@ -2,8 +2,7 @@
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
     "version": "0.1.0",
     "changelog": {
-        "0.1.0": "complete the model package",
-        "0.0.1": "initialize the model package structure"
+        "0.1.0": "complete the model package"
```

Comment:
Hi @tangy5 , here the version number and changelog are still not changed. It should be updated into a new version.